pallets/order-book/src/lib.rs

```diff
@@ -449,8 +449,8 @@ pub mod pallet {
         MaxLotSizeIsMoreThanTotalSupply,
         /// Indicated limit for slippage has not been met during transaction execution.
         SlippageLimitExceeded,
-        /// NFT order books are temporarily forbidden
-        NftOrderBooksAreTemporarilyForbidden,
+        /// Market orders are allowed only for indivisible assets
```

Comment:
Maybe write that instead it should be done through `liquidity_proxy` and namely `LiquiditySource::exchange` for divisible assets? Otherwise it seems quite confusing.
Something like
```suggestion
        /// Market orders are allowed only for indivisible assets.
        /// Divisible assets are traded through `swap` from `liquidity_proxy` pallet, which uses
        /// `LiquiditySource::exchange`.
```

Reply:
User should't know about internal `LiquiditySource::exchange`.  And there is a documentation where it is already should be described.

Reply:
> And there is a documentation where it is already should be described.

Where is it? 🤔